samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/Bridge/Program.cs

```diff
@@ -1,69 +1,39 @@
+        var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus_ConnectionString");
         if (string.IsNullOrWhiteSpace(connectionString))
         {
             throw new Exception("Could not read the 'AzureServiceBus.ConnectionString' environment variable. Check the sample prerequisites.");
         }
 
-        #region bridge-general-configuration
-
-        var bridgeConfiguration = new RouterConfiguration("Bridge");
-        var azureInterface = bridgeConfiguration.AddInterface<AzureServiceBusTransport>("ASB", transport =>
-        {
-            //Prevents ASB from using TransactionScope
-            transport.Transactions(TransportTransactionMode.ReceiveOnly);
-            transport.ConnectionString(connectionString);
-        });
-        var msmqInterface = bridgeConfiguration.AddInterface<MsmqTransport>("MSMQ", transport =>
-        {
-            transport.Transactions(TransportTransactionMode.ReceiveOnly);
-        });
-        msmqInterface.EnableMessageDrivenPublishSubscribe(new InMemorySubscriptionStorage());
-
-        //Configure the host of the MSMQ endpoint
-        msmqInterface.EndpointInstances.AddOrReplaceInstances("publishers", new List<EndpointInstance>
-        {
-            new EndpointInstance("Samples.Azure.ServiceBus.MsmqEndpoint").AtMachine(Environment.MachineName)
-        });
-
-        bridgeConfiguration.AutoCreateQueues();
-
-        var staticRouting = bridgeConfiguration.UseStaticRoutingProtocol();
-
-        staticRouting.AddForwardRoute(
-            incomingInterface: "MSMQ",
-            outgoingInterface: "ASB");
-
-        staticRouting.AddForwardRoute(
-            incomingInterface: "ASB",
-            outgoingInterface: "MSMQ");
-
-        #endregion
-
-        #region bridge-execution
-
-        var bridge = Router.Create(bridgeConfiguration);
-
-        await bridge.Start().ConfigureAwait(false);
-
-        Console.WriteLine("Press any key to exit");
-        Console.ReadKey();
-
-        await bridge.Stop().ConfigureAwait(false);
-
-
-        #endregion
+        await Host.CreateDefaultBuilder()
+            .UseNServiceBusBridge((ctx, bridgeConfiguration) =>
+            {
+                var asbBridgeTransport = new BridgeTransport(new AzureServiceBusTransport(connectionString));
+                asbBridgeTransport.AutoCreateQueues = true;
+                var asbEndpoint = new BridgeEndpoint("Samples.Azure.ServiceBus.AsbEndpoint");
+                asbEndpoint.RegisterPublisher<MyEvent>("Samples.Azure.ServiceBus.MsmqEndpoint");
+                asbBridgeTransport.HasEndpoint(asbEndpoint);
+
+                var msmqBridgeTransport = new BridgeTransport(new MsmqTransport());
+                msmqBridgeTransport.AutoCreateQueues = true;
+                var msmqEndpoint = new BridgeEndpoint("Samples.Azure.ServiceBus.MsmqEndpoint", $"Samples.Azure.ServiceBus.MsmqEndpoint@{Environment.MachineName}");
```

Comment:
Is setting the machine name needed? (our acceptance tests don't need to do this?)

Reply:
For these samples, if we don't include the machine name, the MSMQ endpoint receives the command from the ASB endpoint, but the subsequent publish event from the MSMQ endpoint is never handled by the ASB endpoint.